styleguide.config.js

```diff
@@ -22,6 +23,28 @@ module.exports = {
   getExampleFilename(componentPath) {
     return componentPath.replace(/\.js$/, '.md')
   },
+
+  // This callback allows us to display the source code of a file within markdown:
+  // ```js { "file": "../mySourceCode.js" }
+  // Reference:
+  // https://react-styleguidist.js.org/docs/cookbook.html#how-to-display-the-source-code-of-any-file
+  updateExample(props, exampleFilePath) {
```

Comment:
Allows https://github.com/getethos/ethos-design-system/pull/175/commits/56a35489bfb82e31cdd605569ae45688715a30d6#diff-d94af5a1dbc45e3bc43a4c03b164774dR14 to work in pulling in the entire file as a code example